pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -11,23 +11,22 @@ use confidential_asset::{
     EncryptedAssetIdWrapper, InitializedAssetTxWrapper, MercatAccountId, PubAccountTxWrapper,
 };
 use core::convert::{TryFrom, TryInto};
-use cryptography::{
+use cryptography_core::{
     asset_proofs::{CommitmentWitness, ElgamalSecretKey},
-    mercat::{
-        account::{convert_asset_ids, AccountCreator},
-        asset::AssetIssuer,
-        transaction::{CtxMediator, CtxReceiver, CtxSender},
-        Account, AccountCreatorInitializer, AssetTransactionIssuer, EncryptedAmount,
-        EncryptionKeys, FinalizedTransferTx, InitializedTransferTx, PubAccount, PubAccountTx,
-        SecAccount, TransferTransactionMediator, TransferTransactionReceiver,
-        TransferTransactionSender,
-    },
+    curve25519_dalek::scalar::Scalar,
```

Comment:
I think we should access these via the `mercat` dependency at line 22 now. In order to bundle `cryptography`'s dependencies in one place.